### PR TITLE
motivation

### PR DESCRIPTION
**Pull request by marqh**

### CODE REVIEW

File: course_content/notebooks/numpy_intro.ipynb [hunk 1]
```diff
@@ -24,7 +24,7 @@
    "source": [
     "## Workshop Objectives\n",
     "\n",
-    "At the end of this workshop you will be able to:\n",
+    "The aim of this workshop is to enable you to:\n",
```
Comment (dkillick) [inline course_content/notebooks/numpy_intro.ipynb]: This wording is very atypical of a statement of objectives, which is why I changed it. I am not happy with this wording.

File: course_content/notebooks/numpy_intro.ipynb [hunk 2]
```diff
@@ -95,11 +95,63 @@
    "source": [
     "### Documentation\n",
     "Here is a link to the NumPy documenetation for v1.11:\n",
-    "https://docs.scipy.org/doc/numpy-1.11.0/reference/"
+    "https://docs.scipy.org/doc/numpy-1.11.0/reference/\n",
+    "\n",
+    "### Create an Array\n",
+    "\n",
+    "There are many ways to create a numpy array, such as:"
```
Comment (dkillick) [inline course_content/notebooks/numpy_intro.ipynb]: I wonder whether showing more than on way would be good here. Then delegates can try out the ways they've been shown for themselves as well as going and finding still other ways of creating arrays.

File: course_content/notebooks/numpy_intro.ipynb [hunk 3]
```diff
@@ -126,21 +178,40 @@
...
    "source": [
-    "### Calculation with Broadcasting"
+    "### Fast Calculations\n",
+    "\n",
+    "\n",
+    "NumPy provides routines for fast operations on arrays, including mathematical, logical, shape manipulation, sorting, selecting, I/O, discrete Fourier transforms, basic linear algebra, basic statistical operations, random simulation and much more.\n",
+    "\n",
+    "It is a lot faster than Python alone for numerical computing tasks.\n",
+    "\n",
+    "Element-by-element operations are the “default mode” when an ndarray is involved, but the element-by-element operation is speedily executed by pre-compiled C code. "
```
Comment (dkillick) [inline course_content/notebooks/numpy_intro.ipynb]: It's funny, to me the use of 'speedily' here really increases the informality of this sentence...

File: course_content/notebooks/numpy_intro.ipynb (hunk 1 above)
Comment (marqh) [inline course_content/notebooks/numpy_intro.ipynb]: this appears to be the only concern and I prefer using the wording i have used. 

So, i'm going to adopt this for now, as I'm presenting the trial run of this course

File: course_content/notebooks/numpy_intro.ipynb (hunk 2 above)
Comment (marqh) [inline course_content/notebooks/numpy_intro.ipynb]: i'll pick this up in a further PR, I'd like a bit more text here too

File: course_content/notebooks/numpy_intro.ipynb (hunk 1 above)
Comment (dkillick) [inline course_content/notebooks/numpy_intro.ipynb]: @marqh pragmatic...

**Comment by dkillick** [comment]
We can tackle the (small but apparently) thorny issue of the wording used to introduce the workshop aims when we have more time.

Or just change it depending on who is presenting the course :wink:

**Comment by dkillick** [comment]
Thanks @marqh!

**Pull request merged**